ckan/logic/action/create.py

```diff
@@ -915,6 +915,8 @@ def user_create(context, data_dict):
     :type about: string
     :param openid: (optional)
     :type openid: string
+    :param extras: A dict of extra fields attached to this user. (optional)
+    :type extras: dict
```

Comment:
`plugin_extras`: private extra user data belonging to plugins. Only sysadmin users may set this value.